apiserver/defaulticon.go

```diff
@@ -0,0 +1,280 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package apiserver
+
+// defaultIcon holds the default charm icon SVG content.
+// Keep this in sync with the default icon returned by the charm store.
```

Comment:
Perhaps, that would be interesting to implement. Although the default icon never changed, so it could be overkill.